docs/installation.md

```diff
@@ -30,6 +30,12 @@ Determine your base installation option.
 * A **namespace install** only executes workflows in the namespace it is installed in (typically `argo`). Look for `namespace-install.yaml` in the [release assets](https://github.com/argoproj/argo-workflows/releases/latest).
 * A **managed namespace install**: only executes workflows in a specific namespace ([learn more](managed-namespace.md)).
 
+!!! Warning "namespace install vs. managed namespace install"
+    A namespace install allows Workflows to run only in the namespace where Argo Workflows is installed. A managed namespace install allows Workflows to run only in one namespace besides the one where Argo Workflows is installed. Using a managed namespace install might make sense if you want some users/processes to be able to run Workflows without granting them any privileges in the namespace where Argo Workflows is installed.  
```

Comment:
```suggestion
    A namespace install allows Workflows to run only in the namespace where Argo Workflows is installed.
    A managed namespace install allows Workflows to run only in one namespace _other than_ the one where Argo Workflows is installed.
    You can use a managed namespace install if you want some users or services to run Workflows without granting them privileges in the namespace where Argo Workflows is installed.  
```

- [Use simpler & more direct language](https://kubernetes.io/docs/contribute/style/style-guide/#use-simple-and-direct-language)
- "processes" -> "services", so as to not confuse folks with Unix processes. "Service" as in "ServiceAccount" fits better in k8s
- tiny formatting changes

Reply:
The last sentence here can be moved to `managed-namespace.md`. Could put it in the same paragraph as the example that is to be moved.

The first two sentences I think are redundant with `managed-namespace.md`